packages/presentation/lib/l10n/app_en.arb

```diff
@@ -0,0 +1,6 @@
+{
+    "starMovie": "Star Movie",
```

Comment:
I think u should to replace you hardcoded strings in  details screen. Place all of hardcoded strings to this file. Use S.of(context).string to add string in to your widgets